src/Annotations/src/AnnotationLocator.php

```diff
@@ -32,12 +33,12 @@ final class AnnotationLocator implements SingletonInterface
     /**
      * AnnotationLocator constructor.
      *
-     * @param ClassesInterface      $classLocator
-     * @param AnnotationReader|null $reader
+     * @param ClassesInterface     $classLocator
+     * @param ReaderInterface|null $reader
      *
      * @throws \Doctrine\Common\Annotations\AnnotationException
      */
-    public function __construct(ClassesInterface $classLocator, AnnotationReader $reader = null)
+    public function __construct(ClassesInterface $classLocator, ReaderInterface $reader = null)
     {
         $this->classLocator = $classLocator;
         $this->reader = $reader ?? new AnnotationReader();
```

Comment:
In this case, I would replace the `AnnotationReader` with a `SelectiveReader` one or `(new \Spiral\Attributes\Factory())->create()`, which uses both attributes and annotations.

Reply:
Fixed